firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java

```diff
@@ -46,12 +47,14 @@ private SetOptions(boolean merge, @Nullable FieldMask fieldMask) {
   }
 
   /** @hide */
+  @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Similar to above, this method is no longer public (but note that getFIeldMask() still remains as a public method.)

Reply:
Done